core/src/scene/sceneLoader.cpp

```diff
@@ -1049,6 +1024,69 @@ std::vector<StyleParam> SceneLoader::parseStyleParams(Node params, Scene& scene,
     return out;
 }
 
+StyleUniforms SceneLoader::parseStyleUniforms(const Node& value, Scene& scene) {
+    std::string type = "";
+    std::vector<UniformValue> uniformValues;
+    if (value.IsScalar()) { //float, int (bool), string (texture)
+        UniformValue uVal;
+        try {
+            uVal = value.as<float>();
+            type = "float";
+        } catch (const BadConversion& e) {
+            auto strVal = value.as<std::string>();
+            type = "int";
```

Comment:
cpp-yaml converts quite a lot of things to boolean:  {"y", "n"}, {"yes", "no"}, {"true", "false"}, {"on", "off"} http://yaml.org/type/bool.html. Though chances are low that someone names textures like this :)

@bcamper don't need to doesn't mean we may not add support for int uniforms while we are at it. Or is it about strict feature parity?